client/landing/stepper/declarative-flow/internals/steps-repository/components/accent-color-control/index.tsx

```diff
@@ -127,19 +148,27 @@ const AccentColorControl = ( { accentColor, setAccentColor }: AccentColorControl
 						? __( 'Favorite color' )
 						: __( 'Accent color' ) }
 				</FormLabel>
-
 				<SelectDropdown
 					ref={ accentColorRef }
-					// @ts-expect-error SelectDropdown is defined in .jsx file and has no type definitions generated
```

Comment:
Happy to see this go away 😃

Reply:
Actually it looks like this may still be needed. See the reason for the unit tests failing.
![Screen Shot 2022-11-23 at 13 06 29](https://user-images.githubusercontent.com/2019970/203542945-bd909765-12eb-44ad-b5c0-23af51535289.png)